src/settings/base.py

```diff
@@ -30,29 +28,28 @@
 
 ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
 
-
 # Application definition
-
 INSTALLED_APPS = [
-	'homepage.apps.HomepageConfig',
-	'login.apps.LoginConfig',
-	'sales.apps.SalesConfig',
-	'django.contrib.admin',
-	'django.contrib.auth',
-	'django.contrib.contenttypes',
-	'django.contrib.sessions',
-	'django.contrib.messages',
-	'django.contrib.staticfiles',
+    'homepage.apps.HomepageConfig',
```

Comment:
Try to order this or by alphabetical order or (default, third partys and your own apps) order.